docs/migrating-v2.md

```diff
@@ -56,6 +58,22 @@ import { FormikActions } from 'formik';
 import { FormikHelpers as FormikActions } from 'formik';
 ```
 
+#### `FieldProps`
+
+**`FieldProps` now accepts two generic type parameters.** Both are optional but `FormValues` needs to be passed as the second parameter.
```

Comment:
"Both are optional but `FormValues` has been moved from the first to the second parameter."

The wording currently sounds like "it's optional but _needs_ to be passed."

Reply:
Would be great to elaborate what `FieldValue` means

Reply:
@johrom: absolutely, I'll update.

@jorilallo: I agree it would be good, but I think it would be better to link into the TypeScript docs. However, it doesn't contain a description of those types.

Reply:
@mhienle to be clear, I was going through the docs and I had no idea what `FieldValue` meant, yet alone that it's from TS definitions itself, so an example would not hurt. Same for the actual docs, there's not a single example